airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -111,6 +111,7 @@ def read(
                     )
 
                 try:
+                    self._apply_log_level_to_stream_logger(logger, stream_instance)
```

Comment:
can we remove the apply_log call from [line 166](https://github.com/airbytehq/airbyte/pull/31368/files#diff-51c861828e0a614ef6ee21390f3ba17d7138549f0575051fdb24868ee05eabbbR166)?

Reply:
yep, I thought we could use the `_read_stream` method somewhere else, but I checked and it's only used in the `read` method